src/include/rbd/librbd.hpp

```diff
@@ -410,6 +410,7 @@ class CEPH_RBD_API RBD
   int group_create(IoCtx& io_ctx, const char *group_name);
   int group_remove(IoCtx& io_ctx, const char *group_name);
   int group_list(IoCtx& io_ctx, std::vector<std::string> *names);
+  int group_info(IoCtx& io_ctx, std::string &name, std::string *id);
```

Comment:
There is an existing API with a very similar purpose:
```
int get_group(group_info_t *group_info, size_t group_info_size);
```
I'd rather extend `group_info_t` to also return the group ID.

Reply:
`int get_group(group_info_t *group_info, size_t group_info_size);` belong to class `Image`, it need image info 🤔

Reply:
You are right -- I got confused.  So a new API makes sense, but how about naming it `rbd_group_get_id` for C and `RBD::group_get_id` for C++?  This would make it similar to `rbd_get_id` and `Image::get_id` for images.

A related comment I had is that it would be nice to also add a C API (see `src/include/rbd/librbd.h`) and then a Python binding based on that (see `src/pybind/rbd`).

Reply:
nice, I will implement it